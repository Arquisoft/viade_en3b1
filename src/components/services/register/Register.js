import React from 'react';
import Avatar from '@material-ui/core/Avatar';
import Button from '@material-ui/core/Button';
import CssBaseline from '@material-ui/core/CssBaseline';
import AccountCircleIcon from '@material-ui/icons/AccountCircle';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';
import Container from '@material-ui/core/Container';
import NavBar from '../../graphic interface/NavBar.js';
import Card from '@material-ui/core/Card';
import ContactsOutlinedIcon from '@material-ui/icons/ContactsOutlined';
import ContactsIcon from '@material-ui/icons/Contacts';
import EmojiPeopleIcon from '@material-ui/icons/EmojiPeople';

const useStyles = makeStyles((theme) => ({
    root: {
        minWidth: 275,
        margin: 50,
    },
    paper: {
        marginTop: theme.spacing(5),
        marginBottom: theme.spacing(5),
        display: 'flex',
        flexDirection: 'column',
        alignItems: 'center',
    },
    avatar: {
        margin: theme.spacing(1),
        backgroundColor: theme.palette.warning.main,
    },
    form: {
        width: '80%',
        marginTop: theme.spacing(1),
        alignItems: 'center',
        justifyContent: 'center',
    },
    submit: {
        margin: theme.spacing(1, 0, 2),
    },
    link:{
        margin: theme.spacing(1, 0, 2),
    }
}));

export default function Register() {
    const classes = useStyles();

    return (
        <div>
            <NavBar />
            <Container component="main" maxWidth="sm">
                <Card className={classes.root} elevation={4}>
                    <CssBaseline />
                    <div className={classes.paper}>
                        <Avatar className={classes.avatar}>
                            <AccountCircleIcon />
                        </Avatar>
                        <Typography component="h1" variant="h5">Hi! Welcome to Solid.</Typography>
                        <div className={classes.form}>
                             
                            <Button
                                fullWidth
                                color="primary"
                                className={classes.link}
                                startIcon={<EmojiPeopleIcon/>}
                                onClick={() => window.open('https://solid.inrupt.com/how-it-works')}
                            >How to choose a Provider?</Button>

                            <Button
                                fullWidth
                                variant="outlined"
                                color="primary"
                                className={classes.submit}
                                startIcon={<ContactsOutlinedIcon/>}
                                onClick={() => window.open('https://inrupt.net/register')}
                            >Inrupt</Button>

                            <Button
                                fullWidth
                                variant="outlined"
                                color="primary"
                                className={classes.submit}
                                startIcon={<ContactsIcon/>}
                                onClick={() => window.open('https://solid.community/register')}
                            >Solid Community</Button>

                        </div>
                    </div>
                </Card>
            </Container>
        </div>
    );
}