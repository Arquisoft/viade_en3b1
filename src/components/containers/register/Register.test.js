import Register from "./Register";
import { Avatar } from "@material-ui/core";
import { shallow } from "enzyme";


test("Define Register", () => {
    expect(Register).toBeDefined();
});

test("Renders correctly", () => {
    const wrapper = render(<Register />);
    expect(wrapper).toMatchSnapshot();
});

test("Providers Btns", () => {
    const wrapper = mount(<Register />);

    const element = wrapper.find({id: "btn-inrupt-provider"}).first();
    expect(element).toBeDefined();
    expect(element.text()).toEqual("Inrupt");

    const element2 = wrapper.find({id: "btn-solid-community-provider"}).first();
    expect(element2).toBeDefined();
    expect(element2.text()).toEqual("Solid Community");
});

test("Avatar", () => {
    const wrapper = mount(<Register />);

    const element = wrapper.find(Avatar).first();
    expect(element).toBeDefined();
});;