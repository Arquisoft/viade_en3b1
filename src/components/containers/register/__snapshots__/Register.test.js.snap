// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="makeStyles-root-7"
        >
          <header
            class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
          >
            <div
              class="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-8 MuiToolbar-gutters"
            >
              <div
                class="MuiGrid-root makeStyles-link-10"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineNone MuiTypography-colorInherit"
                    href="#/home"
                  >
                    <div
                      class="MuiGrid-root MuiGrid-container"
                    >
                      <div
                        class="MuiGrid-root MuiGrid-item"
                      >
                        <img
                          alt="Viade Logo"
                          src="logo_2.svg"
                          style="height: 2.5rem; width: 2.5rem;"
                        />
                      </div>
                      <div
                        class="MuiGrid-root MuiGrid-item"
                      >
                        <h4
                          class="MuiTypography-root MuiTypography-h4"
                        >
                          ViaDe
                        </h4>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <a
                aria-disabled="false"
                class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-menuButton-9 MuiButton-containedPrimary"
                href="#/register"
                tabindex="0"
              >
                <span
                  class="MuiButton-label"
                >
                  Sign Up
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </a>
              <a
                aria-disabled="false"
                class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-menuButton-9 MuiButton-colorInherit"
                data-testid="btn-login-navbar"
                href="#/login"
                tabindex="0"
              >
                <span
                  class="MuiButton-label"
                >
                  Sign In
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </a>
            </div>
          </header>
        </div>
        <main
          class="MuiContainer-root MuiContainer-maxWidthSm"
        >
          <div
            class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation4 MuiPaper-rounded"
          >
            <div
              class="makeStyles-paper-2"
            >
              <div
                class="MuiAvatar-root MuiAvatar-circle makeStyles-avatar-3 MuiAvatar-colorDefault"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                  />
                </svg>
              </div>
              <h1
                class="MuiTypography-root MuiTypography-h5"
              >
                Hi! Welcome to Solid.
              </h1>
              <div
                class="makeStyles-form-4"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-link-6 MuiButton-textPrimary MuiButton-fullWidth"
                  data-testid="btn-how-to-choose-provider"
                  id="btn-how-to-choose-provider"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="MuiButton-startIcon MuiButton-iconSizeMedium"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <circle
                          cx="12"
                          cy="4"
                          r="2"
                        />
                        <path
                          d="M15.89 8.11C15.5 7.72 14.83 7 13.53 7h-2.54C8.24 6.99 6 4.75 6 2H4c0 3.16 2.11 5.84 5 6.71V22h2v-6h2v6h2V10.05L18.95 14l1.41-1.41-4.47-4.48z"
                        />
                      </svg>
                    </span>
                    How to choose a Provider?
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-submit-5 MuiButton-outlinedPrimary MuiButton-fullWidth"
                  data-testid="btn-inrupt-provider"
                  id="btn-inrupt-provider"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="MuiButton-startIcon MuiButton-iconSizeMedium"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 4H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V6h16v12zM4 0h16v2H4zm0 22h16v2H4zm8-10c1.38 0 2.5-1.12 2.5-2.5S13.38 7 12 7 9.5 8.12 9.5 9.5 10.62 12 12 12zm0-3.5c.55 0 1 .45 1 1s-.45 1-1 1-1-.45-1-1 .45-1 1-1zm5 7.49C17 13.9 13.69 13 12 13s-5 .9-5 2.99V17h10v-1.01zm-8.19-.49c.61-.52 2.03-1 3.19-1 1.17 0 2.59.48 3.2 1H8.81z"
                        />
                      </svg>
                    </span>
                    Inrupt
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-submit-5 MuiButton-outlinedPrimary MuiButton-fullWidth"
                  data-testid="btn-solid-community-provider"
                  id="btn-solid-community-provider"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="MuiButton-startIcon MuiButton-iconSizeMedium"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 0H4v2h16V0zM4 24h16v-2H4v2zM20 4H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm-8 2.75c1.24 0 2.25 1.01 2.25 2.25s-1.01 2.25-2.25 2.25S9.75 10.24 9.75 9 10.76 6.75 12 6.75zM17 17H7v-1.5c0-1.67 3.33-2.5 5-2.5s5 .83 5 2.5V17z"
                        />
                      </svg>
                    </span>
                    Solid Community
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
        </main>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12"
          style="margin-top: 36rem;"
        >
          <div>
            <footer
              class="makeStyles-footer-252"
            >
              <div
                class="MuiContainer-root MuiContainer-maxWidthXs"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-5"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item"
                  >
                    <h6
                      class="MuiTypography-root MuiTypography-h6"
                    >
                      ViaDe En3B1
                    </h6>
                    <h6
                      class="MuiTypography-root MuiTypography-subtitle2 MuiTypography-colorTextSecondary"
                    >
                      Made by 
                      <a
                        class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                        href="https://www.linkedin.com/in/pablocanalsuarez/"
                        style="color: rgb(48, 44, 88);"
                        target="_blank"
                      >
                        Pablo Ca√±al
                      </a>
                    </h6>
                    <div>
                      <a
                        class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                        href="https://www.linkedin.com/in/pablocanalsuarez/"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                          />
                        </svg>
                      </a>
                      <a
                        class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                        href="mailto:uo264903@uniovi.es"
                        style="margin-left: 0.5rem;"
                        target="_blank"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"
                          />
                        </svg>
                      </a>
                    </div>
                  </div>
                  <hr
                    class="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
                  />
                  <div
                    class="MuiGrid-root MuiGrid-item"
                    style="margin-top: 1rem;"
                  >
                    <a
                      class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                      href="https://github.com/Arquisoft/viade_en3b1/"
                      target="_blank"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                        />
                      </svg>
                    </a>
                  </div>
                </div>
              </div>
            </footer>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="makeStyles-root-7"
      >
        <header
          class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
        >
          <div
            class="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-8 MuiToolbar-gutters"
          >
            <div
              class="MuiGrid-root makeStyles-link-10"
            >
              <div
                class="MuiGrid-root MuiGrid-container"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineNone MuiTypography-colorInherit"
                  href="#/home"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      class="MuiGrid-root MuiGrid-item"
                    >
                      <img
                        alt="Viade Logo"
                        src="logo_2.svg"
                        style="height: 2.5rem; width: 2.5rem;"
                      />
                    </div>
                    <div
                      class="MuiGrid-root MuiGrid-item"
                    >
                      <h4
                        class="MuiTypography-root MuiTypography-h4"
                      >
                        ViaDe
                      </h4>
                    </div>
                  </div>
                </a>
              </div>
            </div>
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-menuButton-9 MuiButton-containedPrimary"
              href="#/register"
              tabindex="0"
            >
              <span
                class="MuiButton-label"
              >
                Sign Up
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-menuButton-9 MuiButton-colorInherit"
              data-testid="btn-login-navbar"
              href="#/login"
              tabindex="0"
            >
              <span
                class="MuiButton-label"
              >
                Sign In
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </header>
      </div>
      <main
        class="MuiContainer-root MuiContainer-maxWidthSm"
      >
        <div
          class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation4 MuiPaper-rounded"
        >
          <div
            class="makeStyles-paper-2"
          >
            <div
              class="MuiAvatar-root MuiAvatar-circle makeStyles-avatar-3 MuiAvatar-colorDefault"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                />
              </svg>
            </div>
            <h1
              class="MuiTypography-root MuiTypography-h5"
            >
              Hi! Welcome to Solid.
            </h1>
            <div
              class="makeStyles-form-4"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-link-6 MuiButton-textPrimary MuiButton-fullWidth"
                data-testid="btn-how-to-choose-provider"
                id="btn-how-to-choose-provider"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  <span
                    class="MuiButton-startIcon MuiButton-iconSizeMedium"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <circle
                        cx="12"
                        cy="4"
                        r="2"
                      />
                      <path
                        d="M15.89 8.11C15.5 7.72 14.83 7 13.53 7h-2.54C8.24 6.99 6 4.75 6 2H4c0 3.16 2.11 5.84 5 6.71V22h2v-6h2v6h2V10.05L18.95 14l1.41-1.41-4.47-4.48z"
                      />
                    </svg>
                  </span>
                  How to choose a Provider?
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-submit-5 MuiButton-outlinedPrimary MuiButton-fullWidth"
                data-testid="btn-inrupt-provider"
                id="btn-inrupt-provider"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  <span
                    class="MuiButton-startIcon MuiButton-iconSizeMedium"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 4H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V6h16v12zM4 0h16v2H4zm0 22h16v2H4zm8-10c1.38 0 2.5-1.12 2.5-2.5S13.38 7 12 7 9.5 8.12 9.5 9.5 10.62 12 12 12zm0-3.5c.55 0 1 .45 1 1s-.45 1-1 1-1-.45-1-1 .45-1 1-1zm5 7.49C17 13.9 13.69 13 12 13s-5 .9-5 2.99V17h10v-1.01zm-8.19-.49c.61-.52 2.03-1 3.19-1 1.17 0 2.59.48 3.2 1H8.81z"
                      />
                    </svg>
                  </span>
                  Inrupt
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-submit-5 MuiButton-outlinedPrimary MuiButton-fullWidth"
                data-testid="btn-solid-community-provider"
                id="btn-solid-community-provider"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  <span
                    class="MuiButton-startIcon MuiButton-iconSizeMedium"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 0H4v2h16V0zM4 24h16v-2H4v2zM20 4H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm-8 2.75c1.24 0 2.25 1.01 2.25 2.25s-1.01 2.25-2.25 2.25S9.75 10.24 9.75 9 10.76 6.75 12 6.75zM17 17H7v-1.5c0-1.67 3.33-2.5 5-2.5s5 .83 5 2.5V17z"
                      />
                    </svg>
                  </span>
                  Solid Community
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </main>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12"
        style="margin-top: 36rem;"
      >
        <div>
          <footer
            class="makeStyles-footer-252"
          >
            <div
              class="MuiContainer-root MuiContainer-maxWidthXs"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-5"
              >
                <div
                  class="MuiGrid-root MuiGrid-item"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-h6"
                  >
                    ViaDe En3B1
                  </h6>
                  <h6
                    class="MuiTypography-root MuiTypography-subtitle2 MuiTypography-colorTextSecondary"
                  >
                    Made by 
                    <a
                      class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                      href="https://www.linkedin.com/in/pablocanalsuarez/"
                      style="color: rgb(48, 44, 88);"
                      target="_blank"
                    >
                      Pablo Ca√±al
                    </a>
                  </h6>
                  <div>
                    <a
                      class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                      href="https://www.linkedin.com/in/pablocanalsuarez/"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                        />
                      </svg>
                    </a>
                    <a
                      class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                      href="mailto:uo264903@uniovi.es"
                      style="margin-left: 0.5rem;"
                      target="_blank"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"
                        />
                      </svg>
                    </a>
                  </div>
                </div>
                <hr
                  class="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
                />
                <div
                  class="MuiGrid-root MuiGrid-item"
                  style="margin-top: 1rem;"
                >
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="https://github.com/Arquisoft/viade_en3b1/"
                    target="_blank"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </div>
          </footer>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
